src/App.vue

```diff
@@ -39,6 +43,12 @@ export default {
       localStorage.setItem("todos", JSON.stringify(todosLocalStorage));
       this.todos.splice(key, 1);
     },
+    setCompleteTaks: function(key) {
+      let todosLocalStorage = JSON.parse(localStorage.getItem("todos"));
+      todosLocalStorage.splice(key, 1, "completed");
```

Comment:
Damit ersetzt du das Array (z. B. ["test", "uncompleted"]) an dem übergebenen key mit dem String "completed"
Also `["test", "uncompleted"]` wird zu `"completed".`
Wenn du dir die Items aus dem Localstorage holst und sie parst bekommst du ein Array, welches Arrays enthält in der Form:
`[
["Duschen","uncompleted"],["Hände waschen","uncompleted"]
]`